database/migrations/model/Version20240419161537.php

```diff
@@ -0,0 +1,50 @@
+<?php namespace Database\Migrations\Model;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Builder;
+use LaravelDoctrine\Migrations\Schema\Table;
+
+/**
+ * Class Version20240419161537
+ * @package Database\Migrations\Model
+ */
+final class Version20240419161537 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $builder = new Builder($schema);
+        if($builder->hasTable("UserStoryDO") && !$builder->hasColumns("UserStoryDO", ["MillionCoreClub"])) {
+            $builder->table("UserStoryDO", function (Table $table) {
+                $table->boolean("MillionCoreClub")->setDefault(false);
```

Comment:
this already exist on data model there is no need of this migration
check the active record definition
https://github.com/OpenStackweb/openstack-org/blob/ebca0d466b5bf217107563f8cacc217ec58d4f75/user-stories/code/infrastructure/active_records/UserStoryDO.php#L24